packages/generator/README.md

```diff
@@ -21,3 +27,130 @@ For installation and usage, see [Measurement Plug-In SDK for Python (`ni-measure
 - [click >= 8.1.3](https://pypi.org/project/click/8.1.3/)
 
 ---
+
+## Developing Measurements: Quick Start
+
+For installation and usage, see [Measurement Plug-In SDK Service for Python - Developing Measurements: Quick Start](https://pypi.org/project/ni_measurement_plugin_sdk_service/#developing-measurements-quick-start) section.
+
+---
+
+## Generating Measurement Clients: Quick Start
+
+This section provides instructions to generate custom measurement clients in Python using Measurement Plug-In SDK for Python.
+
+### Installation
+
+Make sure the system has the recommended Python version is installed. Install Measurement Plug-In SDK Service for Python using [pip](https://pip.pypa.io/).
+
+``` cmd
+REM Activate the required virtual environment if any.
+pip install ni-measurement-plugin-sdk-service
+```
+
+Check if you have installed the expected version of Measurement Plug-In SDK Service for Python installed by running the below command:
+
+```cmd
+pip show ni-measurement-plugin-sdk-service
+```
+
+### Generating a Minimal Python Measurement Client
+
+1. Install the `ni-measurement-plugin-sdk-generator` package.
+
+    ``` cmd
+    REM Activate the required virtual environment if any.
+    pip install ni-measurement-plugin-sdk-generator
+    ```
+
+2. Run the `ni-measurement-plugin-client-generator` tool. Use any of the command line arguments below to create Python measurement clients.
+
+    1. Run this command with optional arguments to create measurement clients for specific measurements.
+
+        ```ni-measurement-plugin-client-generator --measurement-service-class "ni.examples.SampleMeasurement_Python" --module-name "sample_measurement_client"  --class-name "SampleMeasurementClient" --directory-out <new_path_for_created_files>```
+
+        - `--measurement-service-class` specifies the measurement service class for which the client is being generated.
+
+        #### Optional:
+        - `--module-name` and `--class-name` define the module and class names of the generated client. If not specified, they are derived from the measurement service class name.
+
+        - `--directory-out` specifies the output directory for the generated files. If not specified, files are placed in the current directory.
+        
+        > **Note**: For multiple measurement client generation, `--module-name` and `--class-name` are ignored and derived from the service class of each measurement. So, ensure that the measurement service class name adheres to proper naming conventions.
```

Comment:
"When generating multiple measurement clients" would be more clear than turning it into a noun